### PR TITLE
Update fi.coffee

### PR DESCRIPTION
**Pull request by dandabs**

### CODE REVIEW

File: app/locale/fi.coffee [hunk 1]
```diff
@@ -161,10 +161,10 @@ module.exports = nativeDescription: "suomi", englishDescription: "Finnish", tran
     view_progress: "Tarkastele edistymistäni"
     go_to_courses: "Siirry omiin kursseihin"
     want_coco: "Haluatko CodeCombat:in kouluusi?"
-#    educator: "Educator"
-#    student: "Student"
+    educator: "Opettaja" # {change}
```
Comment (mrfinch) [inline app/locale/fi.coffee:164]: what is # {change} here? Do you intend to change translation or it's a comment? Same question for below changes

**Review by mrfinch — COMMENTED** [review]
Sorry for the delay in reviewing

File: app/locale/fi.coffee (hunk 1 above)
Comment (dandabs) [reply app/locale/fi.coffee]: I must've misinterpreted the translation instructions where it stated to add # {change} when editing the en.coffee file. I've removed those instances and committed again.

File: app/locale/fi.coffee [hunk 2]
```diff
@@ -3050,7 +3050,7 @@ module.exports = nativeDescription: "suomi", englishDescription: "Finnish", tran
 #    learn_at_own_pace: "Learn at Your Own Pace"
 #    monthly_sub: "Monthly Subscription"
 #    buy_now: "Buy now"
-#    per_month: " / mo"
+    per_month: " / kk"
```
Comment (mrfinch) [inline app/locale/fi.coffee:3053]: if you want to enable this key, its parent which is `parents_landing_2` needs to be uncommented by removing `#`

File: app/locale/fi.coffee [hunk 3]
```diff
@@ -3208,7 +3208,7 @@ module.exports = nativeDescription: "suomi", englishDescription: "Finnish", tran
 #  browser_recommendation:
 #    title: "CodeCombat works best on Chrome!"
 #    pitch_body: "For the best CodeCombat experience we recommend using the latest version of Chrome. Download the latest version of chrome by clicking the button below!"
-#    download: "Download Chrome"
+    download: "Lataa Chrome"
```
Comment (mrfinch) [inline app/locale/fi.coffee:3211]: uncomment parent `browser_recommendation`

File: app/locale/fi.coffee [hunk 4]
```diff
@@ -3284,7 +3284,7 @@ module.exports = nativeDescription: "suomi", englishDescription: "Finnish", tran
 #    codecombat_ai_league: "CodeCombat AI League"
 #    register: "Register"
 #    not_registered: "Not Registered"
-#    world: "World"
+    world: "Maailma"
```
Comment (mrfinch) [inline app/locale/fi.coffee:3287]: uncomment `admin`

File: app/locale/fi.coffee [hunk 5]
```diff
@@ -3340,7 +3340,7 @@ module.exports = nativeDescription: "suomi", englishDescription: "Finnish", tran
 
 #  form_validation_errors:
 #    required: "Field is required"
-#    invalidEmail: "Invalid email"
-#    invalidPhone: "Invalid phone number"
-#    emailExists: "Email already exists"
-#    numberGreaterThanZero: "Should be a number greater than 0"
+    invalidEmail: "Väärä sähköpostiosoite"
```
Comment (mrfinch) [inline app/locale/fi.coffee:3343]: uncomment `form_validation_errors`

**Review by mrfinch — COMMENTED** [review]
To enable a key, its parent needs to be enabled too. I have added comments related to it in my review. 

Let me know if you need any help.

Thanks for contributing 😄

**Comment by mrfinch** [comment]
closing because of inactivity, please re-open if needed

**Pull request closed without merging**